### DISCUSSION TITLE
Client awareness of FerretDB

### DISCUSSION THREAD

**Discussion by BoBiene** [FerretDB/FerretDB#3186] [General]
is there a way (or a plan) how a client can identify if it is connected to a mongoDB server or a FerretDB?
And the version of the FerretDB?

**Comment by chilagrow** [comment]
Hello @BoBiene :wave: 

Is `buildInfo` command what you are looking for? We include `ferretdbVersion` field in the response which only exists on FerretDB.
```
test> db.runCommand({buildInfo: {}})
{
  ...
  ferretdbVersion: 'v1.7.0-19-g2f5d9837', 
  ...
}
```

**Reply by BoBiene** [reply]
Yes, thank you. This is exactly what i was looking for.
